.github/workflows/chart-bumper.yaml

```diff
@@ -0,0 +1,64 @@
+name: Upgrade helm charts minor versions
+
+on:
+  workflow_run:
+    workflows: ["Release Charts"]
+    types: [completed]
+
+jobs:
+  upgrade-helm-charts:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.7.2
+
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+
+      - name: install pyaml
+        run: |
+          pip3 install pyaml==21.10.1
+
+      - name: install updateCli
+        run: |
+          curl -sL -o /tmp/updatecli_amd64.deb https://github.com/updatecli/updatecli/releases/download/v0.36.0/updatecli_amd64.deb
+          sudo apt install /tmp/updatecli_amd64.deb
+
+      - name: Set up chart-testing
+        uses: helm/chart-testing-action@v2.2.1
+        with:
+          version: v3.5.1
+
+      - name: Run chart bumper script & update docs
+        run: |
+          python scripts/chart_bumper.py
+          rm -f manifest.yaml
```

Comment:
what are pros/cons of having it as a part of py script?

Reply:
no specific reason why only a python script. The script is pretty simple, it creates a manifest for update-cli and invokes it for all the charts.  Added a py script solely for ease of use.